website/modules/docs/Web.js

```diff
@@ -35,80 +35,80 @@ export default {
     {
       label: 'Basic',
       slug: 'basic',
-      load: require('bundle?lazy!babel!../examples/Basic'),
-      loadSource: require('bundle?lazy!!prismjs?lang=jsx!../examples/Basic.js')
+      load: require('../examples/Basic?bundle'),
+      loadSource: require('../examples/Basic.js?prismjs')
```

Comment:
This configuration has been moved to the Webpack config. Because we are loading one file two different ways, we need to use a resource query to differentiate between when to read it as JavaScript and when to read it as a string to markup.